Projektskabelon-master/src/main/java/FunctionLayer/OverlayMaterialCalculator.java

```diff
@@ -177,6 +180,7 @@ public  static Material screwForOverlayOneWall ( Wall wall, String overlayName)
             fyr.setComment("Vertical framing-dør");
             fyr.setSize(fyrLengths[i]);
             fyr.setAmount(1);
+            fyr.setPrice(LogicFacade.getPrice(fyr.getId()));
```

Comment:
If you set the price here, it will be calculated for the needed size, not for the available one. I calculate de available sizes first in function layer